wall_e/src/resources/cogs/reminders.py

```diff
@@ -13,8 +13,11 @@
 import psycopg2
 # import os
 import datetime
+import pytz
 logger = logging.getLogger('wall_e')
 
+_timezone = pytz.timezone("Canada/Pacific")
```

Comment:
is there a reason you didnt name this something more descriptive like `pacific_timezone`?

Reply:
On the off chance someone wanted to fork the bot for their own use (say a university in Toronto or whatever), they could change it to something more appropriate for them.
Not too attached to this though, so I can change to `pacific_timezone` if necessary

Reply:
then in that case, I say change it to `wall_e_timezone` and add a comment that is basically your comment.

Reply:
Gotcha, will do

Reply:
you did not add the comments

Reply:
Oh right, my bad

Reply:
Added the comments